### DISCUSSION TITLE
The status is always "Submitted"

### DISCUSSION THREAD

**Discussion by hachinoone** [PaddlePaddle/PaddleSpatial#144] [BaiduKDDCup2022] [answered]
![QQ图片20220604123633](https://user-images.githubusercontent.com/26597075/171983282-d03102fa-2209-4829-8f98-c1cdcd765012.png)

Dear Organizer,

Could you please help me find out why the status remains "Submitted"? I have a sucessful submission two hours before the last submission. Thank you very much.

**Comment by ramber1836** [comment, marked as answer]
I have re-evaluated your submission. You can check it now.
